src/EventHub/EventHub.Autorest/custom/Set-AzEventHubNamespace.ps1

```diff
@@ -0,0 +1,345 @@
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Runtime.DefaultInfo(Script = '(Get-AzContext).Subscription.Id')]
+        [System.String]
+        # The ID of the target subscription.
+        ${SubscriptionId},
+
+        [Parameter(HelpMessage = "The Alternate Name of Eventhub Namespace")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${AlternateName},
+
+        [Parameter(HelpMessage = "The ClusterArmId of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${ClusterArmId},
+
+        [Parameter(HelpMessage = "DisableLocalAuth")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${DisableLocalAuth},
+
+        [Parameter(HelpMessage = "EncryptionKeySource")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeySource},
+
+        [Parameter(HelpMessage = "EncryptionKeyVaultProperty.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionKeyVaultProperty},
+
+        [Parameter(HelpMessage = "EncryptionRequireInfrastructureEncryption")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${EncryptionConfig}, #EncryptionConfig
+
+        [Parameter(HelpMessage = "IdentityType of EventHub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IdentityType},
+
+        [Parameter(HelpMessage = "IdentityUserAssigneIdentity")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Collections.Hashtable]
+        ${IdentityId},#IdentityId
+
+        [Parameter(HelpMessage = "IsAutoInflateEnabled")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${IsAutoInflateEnabled},
+
+        [Parameter(HelpMessage = "KafkaEnabled")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${KafkaEnabled},
+
+        [Parameter(HelpMessage = "Location of EventHub Namespace")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${Location},
+
+        [Parameter(HelpMessage = "MaximumThroughputUnit of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.Int64]
+        ${MaximumThroughputUnit},
+
+        [Parameter(HelpMessage = "MinimumTlsVersion of Eventhub Namespace.")]
+        [Microsoft.Azure.PowerShell.Cmdlets.EventHub.Category('Path')]
+        [System.String]
+        ${MinimumTlsVersion},
+
+        [Parameter(HelpMessage = "PrivateEndpointConnectionName.")]
```

Comment:
Remove PrivateEndpointConnection from both New and Set cmdlets. It cannot be set this way.